src/test/java/com/segment/analytics/android/integrations/adobeanalytics/AdobeTest.java

```diff
@@ -617,6 +617,68 @@ public void trackVideoSeekComplete() {
     verify(heartbeat).trackEvent(MediaHeartbeat.Event.SeekComplete, null, null);
   }
 
+  @Test
+  public void trackVideoAdStarted() {
+    Properties properties = new Properties();
```

Comment:
Could we re-use the same object rather than repeat all of these fixtures?

Reply:
Will do.